src/app/candidate-management/candidate-management-detail/candidate-management-detail.component.ts

```diff
@@ -239,7 +239,7 @@ export class CandidateManagementDetailComponent implements OnInit, AfterViewInit
       // call update for all additional authors
       for(let author of authorlist) {
         if(author.userId !== first_author) {
-          this.candidateManagementService.updateAuthorsCandidate(this.candidate, author).subscribe(result => {
+          this.candidateManagementService.updateAuthorsCandidate(result, author).subscribe(result => {
```

Comment:
Rename `result` (e.g. to `candidate` or `createdCandidate` or similar) to avoid having two result variables with different meaning.
Also: Can result be a failure state (e.g. `null`)? If so, then guard against this.